src/components/raycaster.js

```diff
@@ -143,9 +145,10 @@ module.exports.Component = registerComponent('raycaster', {
     });
 
     // Update intersectedEls.
-    intersectedEls = this.intersectedEls = intersections.map(function getEl (intersection) {
-      return intersection.object.el;
-    });
+    intersectedEls.length = intersections.length;
```

Comment:
Do we know the effects of writting the length propery of an array? Are elements disposed if the new length is smaller than the old one?